specs/sql/StatementTypes.md

```diff
@@ -197,6 +197,44 @@ Rows with automatically selected `ROWID`s are guaranteed to have `ROWID`s that h
 
 Note that "monotonically increasing" does not imply that the `ROWID` always increases by exactly one. One is the usual increment. However, if an insert fails due to (for example) a uniqueness constraint, the `ROWID` of the failed insertion attempt might not be reused on subsequent inserts, resulting in gaps in the `ROWID` sequence. Tableland guarantees that automatically chosen `ROWID`s will be increasing but not that they will be sequential.
 
+### ALTER TABLE
```

Comment:
Added this section based on https://www.sqlite.org/lang_altertable.html

The README.md changes was autogenerated by running
```
pandoc -s --toc \
    -t gfm+tex_math_dollars\
    -f gfm+tex_math_dollars \
    -B Header.md \
    StatementTypes.md \
    DataTypes.md \
    Encoding.md \
    -o README.md
 ```